core/src/consensus.rs

```diff
@@ -548,8 +544,7 @@ impl Tower {
         let mut slot_with_ancestors = vec![vote.slot];
         slot_with_ancestors.extend(vote_slot_ancestors.unwrap());
         for slot in slot_with_ancestors {
-            let entry = &mut stake_lockouts.entry(slot).or_default();
-            entry.lockout += vote.lockout();
+            stake_lockouts.insert(slot, StakeLockout::default());
```

Comment:
I think this is replacing the slot entry every time (resets `stake_lockouts.stake`, but it could already exist 😄 )